### PR TITLE
Add Vercel Analytics

### PR DESCRIPTION
**Pull request by ysm-dev**
- https://vercel.com/analytics

examples
- https://vercel.com/
- https://rgllm.com/
- https://www.mailing.run/
- https://lioncityhacks.com/
- https://mario.dev/

### CODE REVIEW

File: src/technologies/v.json [hunk 1]
```diff
@@ -293,6 +293,20 @@
...
+  "Vercel Analytics": {
+    "cats": [
+      10
+    ],
+    "js": {
+      "va": ""
+    },
+    "scriptSrc": "/va/script\\.js",
+    "implies": [
+      "Vercel"
```
Comment (nurbek91) [inline src/technologies/v.json:305]: Maybe replace `"implies"`  to `"requires"`, then probably it will help to avoid fall positive results.

**Review by nurbek91 — COMMENTED** [review]
Hey mate, thanks!!

File: src/technologies/v.json (hunk 1 above)
Comment (ysm-dev) [reply src/technologies/v.json:305]: [e4bf0ef](https://github.com/wappalyzer/wappalyzer/pull/6932/commits/e4bf0ef391e5204bf8bc15abf27256b8d189e7f7)

Thanks!

**Review by nurbek91 — APPROVED** [review]

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**